nipype/pipeline/plugins/legacymultiproc.py

```diff
@@ -64,7 +64,7 @@ def run_node(node, updatehash, taskid):
     # Try and execute the node via node.run()
     try:
         result["result"] = node.run(updatehash=updatehash)
-    except:  # noqa: E722, intendedly catch all here
+    except:
```

Comment:
I think in general we would want to catch cases of E722 instead of blanket ignoring.